### PR TITLE
fix: fix "Allowed roots should not contain '/'" assertion test error

### LINKED ISSUE #749: fix "Allowed roots should not contain '/'" assertion tests error
## What versions of software are you using?
- Operating System:
- IntelliJ version:
- Extension version:
- Are you targeting a kubernetes or an Openshift cluster ? If possible, provide its version:

## What did you run exactly?
```
0K[1m  Test [22mtestQuarkusDevfile[31m FAILED[33m (1.6s)[31m

  com.intellij.testFramework.TestLoggerFactory$TestLoggerAssertionError: Allowed roots should not contain '/'. You can disable roots access check explicitly if you don't need it.
      at com.intellij.testFramework.TestLoggerFactory$TestLogger.error(TestLoggerFactory.java:404)
      at com.intellij.openapi.diagnostic.Logger.error(Logger.java:436)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:431)
      at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1075)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$3(PassExecutorService.java:392)
      at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:610)
      at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:685)
      at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:641)
      at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:609)
      at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:78)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:391)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:367)
      at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:200)
      at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:184)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:365)
      at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:187)
      at java.base@17.0.9/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
      at java.base@17.0.9/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
      at java.base@17.0.9/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
      at java.base@17.0.9/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
      at java.base@17.0.9/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
  Caused by: java.lang.AssertionError: Allowed roots should not contain '/'. You can disable roots access check explicitly if you don't need it.
      at com.intellij.openapi.vfs.newvfs.impl.VfsRootAccess.allowedRoots(VfsRootAccess.java:195)
      at com.intellij.openapi.vfs.newvfs.impl.VfsRootAccess.assertAccessInTests(VfsRootAccess.java:71)
      at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.findInPersistence(VirtualDirectoryImpl.java:182)
      at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.doFindChild(VirtualDirectoryImpl.java:137)
      at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.findChild(VirtualDirectoryImpl.java:83)
      at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.findChild(VirtualDirectoryImpl.java:535)
      at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.findChild(VirtualDirectoryImpl.java:51)
      at com.intellij.openapi.vfs.newvfs.VfsImplUtil.findFileByPath(VfsImplUtil.java:56)
      at com.intellij.openapi.vfs.impl.jar.JarFileSystemImpl.findFileByPath(JarFileSystemImpl.java:111)
      at com.intellij.openapi.vfs.impl.VirtualFileManagerImpl.findByUrl(VirtualFileManagerImpl.java:344)
      at com.intellij.openapi.vfs.impl.VirtualFileManagerImpl.findFileByUrl(VirtualFileManagerImpl.java:331)
      at com.intellij.openapi.vfs.VfsUtil.findFileByURL(VfsUtil.java:189)
      at com.jetbrains.jsonSchema.extension.JsonSchemaProviderFactory.getResourceFile(JsonSchemaProviderFactory.java:47)
      at com.intellij.lang.javascript.EmbeddedJsonSchemaFileProvider.<init>(EmbeddedJsonSchemaFileProvider.java:29)
      at com.intellij.lang.javascript.JavaScriptJsonSchemaProviderFactory.createProvider(JavaScriptJsonSchemaProviderFactory.java:58)
      at com.intellij.lang.javascript.JavaScriptJsonSchemaProviderFactory.createProviders(JavaScriptJsonSchemaProviderFactory.java:52)
      at com.intellij.lang.javascript.JavaScriptJsonSchemaProviderFactory.getProviders(JavaScriptJsonSchemaProviderFactory.java:64)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl$JsonSchemaProviderFactories.getProvidersFromFactories(JsonSchemaServiceImpl.java:646)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl$JsonSchemaProviderFactories.getDumbAwareProvidersAndUpdateRestWhenSmart(JsonSchemaServiceImpl.java:622)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl$JsonSchemaProviderFactories.getProviders(JsonSchemaServiceImpl.java:600)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl.lambda$new$0(JsonSchemaServiceImpl.java:57)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl$MyState.lambda$new$0(JsonSchemaServiceImpl.java:502)
      at com.intellij.util.concurrency.SynchronizedClearableLazy._get_value_$lambda$1$lambda$0(SynchronizedClearableLazy.kt:41)
      at java.base/java.util.concurrent.atomic.AtomicReference.updateAndGet(AtomicReference.java:210)
      at com.intellij.util.concurrency.SynchronizedClearableLazy.getValue(SynchronizedClearableLazy.kt:40)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl.getProvidersForFile(JsonSchemaServiceImpl.java:242)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl.getSchemasForFile(JsonSchemaServiceImpl.java:175)
      at com.jetbrains.jsonSchema.impl.JsonCachedValues.getSchemaFile(JsonCachedValues.java:282)
      at com.jetbrains.jsonSchema.impl.JsonCachedValues.getSchemaFile(JsonCachedValues.java:274)
      at com.jetbrains.jsonSchema.impl.JsonCachedValues.lambda$computeSchemaForFile$7(JsonCachedValues.java:248)
      at com.intellij.psi.util.CachedValuesManager$1.compute(CachedValuesManager.java:158)
      at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:37)
      at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:240)
      at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:43)
      at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:240)
      at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
      at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
      at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:66)
      at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
      at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:27)
      at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:69)
      at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:155)
      at com.jetbrains.jsonSchema.impl.JsonCachedValues.computeSchemaForFile(JsonCachedValues.java:247)
      at com.jetbrains.jsonSchema.impl.JsonSchemaServiceImpl.getSchemaObject(JsonSchemaServiceImpl.java:330)
      at org.jetbrains.yaml.schema.YamlJsonSchemaInspectionBase.buildVisitor(YamlJsonSchemaInspectionBase.java:38)
      at com.intellij.codeInspection.InspectionEngine.createVisitor(InspectionEngine.java:67)
      at com.intellij.codeInsight.daemon.impl.InspectionRunner.createContext(InspectionRunner.java:405)
      at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$inspect$4(InspectionRunner.java:130)
      at com.intellij.codeInspection.InspectionEngine.withSession(InspectionEngine.java:246)
      at com.intellij.codeInsight.daemon.impl.InspectionRunner.inspect(InspectionRunner.java:126)
      at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:113)
      at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:80)
      at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:406)
      at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.runWithSpanIgnoreThrows(trace.kt:76)
      at com.intellij.platform.diagnostic.telemetry.helpers.TraceUtil.runWithSpanThrows(TraceUtil.java:34)
      at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:401)
      ... 18 more
```

### PR DESCRIPTION
**Pull request by sbouchet**
## Was the change discussed in an issue?
fixes #749

### CODE REVIEW

File: src/test/java/org/jboss/tools/intellij/openshift/validation/DevfileSchemasTest.java [hunk 1]
```diff
@@ -11,18 +11,14 @@
...
 import org.jetbrains.yaml.schema.YamlJsonSchemaHighlightingInspection;
 
-import java.io.File;
 
 public class DevfileSchemasTest extends BasePlatformTestCase {
 
     public void testQuarkusDevfile() {
         myFixture.setTestDataPath("src/test/resources");
         myFixture.enableInspections(YamlJsonSchemaHighlightingInspection.class);
-        String path = new File("src").getAbsoluteFile().getParentFile().getAbsolutePath();
```
Comment (adietish) [inline src/test/java/org/jboss/tools/intellij/openshift/validation/DevfileSchemasTest.java:24]: why was this called in the test? It seems to have had no function if it's possible to simply remove it?

**Review by adietish — APPROVED** [review]
Wondering why the code, that we removed, was there in the first place.
Other than this LGTM since tests are now gree.

File: src/test/java/org/jboss/tools/intellij/openshift/validation/DevfileSchemasTest.java (hunk 1 above)
Comment (sbouchet) [reply src/test/java/org/jboss/tools/intellij/openshift/validation/DevfileSchemasTest.java:24]: i really don't know !

**Pull request merged**